flink-examples/flink-examples-table/src/main/scala/org/apache/flink/table/examples/StreamSQLExample.scala

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.flink.examples.scala
+package org.apache.flink.table.examples
```

Comment:
This should be called `org.apache.flink.table.examples.scala` to be consistent with `org.apache.flink.table.examples.java`. IMHO the language should be right after the `table`: `org.apache.flink.table.java.examples`